lightning/src/util/config.rs

```diff
@@ -59,6 +59,14 @@ pub struct ChannelHandshakeConfig {
 	/// Default value: 1. If the value is less than 1, it is ignored and set to 1, as is required
 	/// by the protocol.
 	pub our_htlc_minimum_msat: u64,
+	/// Set to the largest value HTLC we will accept to process.
```

Comment:
Hmm, I don't think this makes sense - we don't actually *enforce* this, so suggesting to users that it is a "real option" seems strange to me. Nor can we enforce this securely, since our counterparty won't enforce it on inbound HTLCs.

Reply:
I'm a little confused about this option then. I guess it is only used to help others with routing via us?
What about this comment:

/// Set to the largest value HTLC which we expect to be routable via our channels.

/// This value is communicated to other nodes at our channel announcements and is
/// supposed to help other nodes to make better routing decisions and reduce failure rate.

Reply:
> I guess it is only used to help others with routing via us?

Yes, I believe that is the intent.

> What about this comment:

I dont think we need to *store* it at all - it can always be calculated on the fly.

Reply:
FWIW, lnd and (IIUC) c-lightning enforce max HTLC, and offer it as a configurable option for users. So, putting it out there that I think it'd make sense for us to do so as well :P

Reply:
I don't feel like this option is particularly useful for LN node operators, so I'm fine with just dropping it for the sake of less code...

Matt's opinion on your suggestion would be useful here I guess.

Reply:
That's a fair point, don't feel too strongly so I'm fine to continue on the course of removing it.